wiki/Guides/How_to_Use_the_Offset_Wizard/id.md

```diff
@@ -0,0 +1,26 @@
+# Bagaimana cara menggunakan Offset Wizard
+
+*Kunjungi juga: [Offset Wizard](/wiki/Options/Offset_Wizard) dan [Universal offset](/wiki/Universal_offset)*
+
+Jika Kamu merasa [hitsound](/wiki/Beatmapping/Hitsound) tidak sinkron dengan musik di setiap beatmap yang dimainkan, kemungkinan besar Kamu perlu mengubah [Universal offset](/wiki/Universal_offset). Cara yang paling mudah untuk melakukan perubahan pada Universal offset adalah dengan menggunakan **Offset Wizard**.
+
+Ikuti langkah-langkah berikut untuk menyesuaikan Universal offset menggunakan Offset Wizard.
+
+## Membukan Offset Wizard
+
+1. Buka klien osu!.
+2. Pergi ke menu `Options`, atau tekan kombinasi `Ctrl` + `O` pada keyboard.
+3. Cari/ketik kata "offset" pada pencarian cepat, dan tekan tombol `Offset wizard`.
+
+Kemudian Offset Wizard akan terbuka, memutar lagu yang sebelumnya diputar di klien osu!, sementara bar hijau bergulir di layar dan metronom berdetak.
+
+## Mencocokkan Universal offset
+
+1. Dengarkan lagu dan metronom secara baik-baik ketika dimainkan bersama.
+2. Mulailah menyesuaikan Universal offset ke atas atau bawah, menggunakan tombol panah yang sesuai, atau roda gulir mouse.
+   - Jika Kamu merasa hitsound dimainkan terlalu awal dalam game, Kamu perlu *mengurangi* Universal offset.
```

Comment:
```suggestion
   - Jika Kamu merasa suara hitsound dimainkan terlalu awal dalam game, Kamu perlu *mengurangi* Universal offset.
```

Reply:
to make things clearer

Reply:
may using without it instead, it's a kind of